core/capabilities/ccip/ccipevm/extradatacodec.go

```diff
@@ -14,11 +14,11 @@ const (
 	evmDestExecDataKey    = "destGasAmount"
 )
 
-// ExtraDataCodec is a concrete implementation of SourceChainExtraDataCodec
-type ExtraDataCodec struct{}
+// EVMExtraDataCodec is a concrete implementation of SourceChainExtraDataCodec
+type EVMExtraDataCodec struct{}
```

Comment:
We don't need the EVM prefix as this type is in specific `ccipevm` package

Reply:
yeah I have this concern but can't think of a better name, was thinking sourceChainExtraDataCodec, but it would be duplicate of the interface it implements.